extensions/ui/org.eclipse.smarthome.ui.paper/web-src/js/controllers.items.js

```diff
@@ -0,0 +1,124 @@
+angular.module('PaperUI.controllers.configuration').controller('ItemSetupController', function($scope, $timeout, $mdDialog, itemService, toastService, sharedProperties) {
+    $scope.setSubtitle([ 'Items' ]);
+    $scope.setHeaderText('Shows all configured Items.');
+    $scope.items = [];
+    $scope.refresh = function() {
+        $scope.items = itemService.getNonRecursiveAll();
+    };
+    $scope.remove = function(item, event) {
+        event.stopImmediatePropagation();
+        $mdDialog.show({
+            controller : 'ItemRemoveController',
+            templateUrl : 'partials/dialog.removeitem.html',
+            targetEvent : event,
+            hasBackdrop : true,
+            locals : {
+                item : item
+            }
+        }).then(function() {
+            $scope.refresh();
+        });
+    }
+    $scope.getSrcURL = function(category, type) {
+        return category ? 'http://localhost:8080/icon/' + category.toLowerCase() : type ? 'http://localhost:8080/icon/' + type.toLowerCase().replace('item', '') : '';
```

Comment:
Is it necessary to use the full URL here? I don't think it is a good idea to force the usage of 8080 as port to have a full functional Paper UI.